pkgs/tools/security/bloodhound-python/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, python3 }:
```

Will create easier to read diffs in the future.

```suggestion
{ lib
, fetchFromGitHub
, python3
}:
```